### ISSUE TITLE
Split low- and high-level itsdb code into separate modules

### ISSUE THREAD

**Issue by goodmami** [delph-in/pydelphin#228]
The `delphin.itsdb` module is pretty huge, even with the deprecated stuff gone. So it could be useful to move the most basic code for working with test suite databases into a `delphin.tsdb` module, leaving the higher-level modeling and processing to `delphin.itsdb`. Specifically the `tsdb` module would have code to:

* read and write relations files
* find the appropriate (text or gzipped) table files
* open and write table files
* decode and encode table lines
* escape and unescape reserved characters

Basically, it only contains code to deal with the details of TSDB's file-based databases, such that alternate implementations of `itsdb` (e.g., regular or using sqlite, see #204) would use this module to interface with the on-disk data.
